{
	"cve": {
		"CVE_data_meta": {
			"ASSIGNER": "cve@mitre.org",
			"ID": "CVE-2024-30692"
		},
		"data_format": "MITRE",
		"data_type": "CVE",
		"data_version": "4.0",
		"description": {
			"description_data": [
				{
					"lang": "en",
					"value": "A issue was discovered in ROS2 Galactic Geochelone versions ROS_VERSION 2 and ROS_PYTHON_VERSION 3, allows remote attackers to cause a denial of service (DoS) in the ROS2 nodes. NOTE: this is disputed by multiple third parties who believe there was not reasonable evidence to determine the existence of a vulnerability."
				}
			]
		},
		"problemtype": {
			"problemtype_data": [
				{}
			]
		},
		"references": {
			"reference_data": [
				{
					"name": "https://github.com/yashpatelphd/CVE-2024-30692",
					"url": "https://github.com/yashpatelphd/CVE-2024-30692"
				}
			]
		}
	},
	"impact": {},
	"configurations": {
		"CVE_data_version": "4.0"
	},
	"lastModifiedDate": "2024-04-17T01:15Z",
	"publishedDate": "2024-04-09T07:15Z"
}
